docs/Reference/Server.md

```diff
@@ -463,7 +463,7 @@ Defines the label used for the request identifier when logging the request.
 <a id="factory-gen-request-id"></a>
 
 Function for generating the request-id. It will receive the incoming request as
```

Comment:
I think here should mention it expect the function is error free. 
Instead of mention is will crash the server if you throw.

Reply:
What @climba03003 said